server/bastionlab_polars/src/composite_plan.rs

```diff
@@ -18,7 +21,15 @@ pub struct CompositePlan(Vec<CompositePlanSegment>);
 #[derive(Debug, Serialize, Deserialize)]
 pub enum CompositePlanSegment {
     PolarsPlanSegment(LogicalPlan),
-    UdfPlanSegment { columns: Vec<String>, udf: String },
+    StringTransformerPlanSegment {
+        columns: Vec<String>,
+        model: String,
+        add_special_tokens: u8,
+    },
+    UdfPlanSegment {
```

Comment:
do we support that? (python & rust side)

Reply:
Do we support what exactly?